script.js

```diff
@@ -0,0 +1,315 @@
+      Array.from(keyboard.children).forEach(key =>{
+        if(key.innerText.length == 1){
+          if(key.classList.contains('uppercase') ){
+            key.classList.remove('uppercase');
+            key.classList.add('lowercase');
+          }else{
+            key.classList.remove('lowercase');
+            key.classList.add('uppercase');
+          }
+        }
+      })
+      capsLockKey.classList.toggle('caps-active');
+    }
+  }
+
+  // Checks if CapsLock is pressed or not
+    //return: capsActive{boolean}: true -  letters in uppercase
+    //                             false - letters in lowercase
+
+  function isCapsActive(){
+    let capsActive = false;
+    if(keyboard.firstChild.classList.contains('uppercase')){
+      capsActive = true;
+    }
+    return capsActive;
+  }
+
+  // Clicking to shift
+    //@e{object} -  addEventListener function event
+    //@keysUnshift{array} - array of letters without shift pressing
+    //@keysShift{array} - array of letters with shift pressing
+
+  function shiftKeyPress(e, keysUnshift, keysShift){
+    if((e.key == 'Shift' && e.repeat == false) || e.target.innerText == 'Shift'){
+      Array.from(keyboard.children).forEach((key, index) =>{
+        if(e.type == 'keydown' || e.type == 'mousedown'){
+          key.innerText = keysShift[index];
+        }else if(e.type == 'keyup' || e.type == 'mouseup'){
+          key.innerText = keysUnshift[index];
+        }
+        if(key.innerText.length == 1){
+          if(key.classList.contains('uppercase') ){
+            key.classList.remove('uppercase');
+            key.classList.add('lowercase');
+          }else{
+            key.classList.remove('lowercase');
+            key.classList.add('uppercase');
+          }
+        }
+      })
+    }
+  }
+
+  // Changes the keyboard's language (eng or rus)
+    //@e{object} -  addEventListener function event
+    //return:  @lang{string}: eng/rus - current keyboard language
+
+  function changeLang(e){
+    if(e.ctrlKey && e.altKey){
+      if(keyboard.classList.contains('eng')){
+        keyboard.classList.add('rus');
+        keyboard.classList.remove('eng');
+        Array.from(keyboard.children).forEach((key, index) =>{
+          key.innerText = keysRus[index];
+        })
+      }else{
+        keyboard.classList.add('eng');
+        keyboard.classList.remove('rus');
+        Array.from(keyboard.children).forEach((key, index) =>{
+          key.innerText = keysEng[index];
+        })
+      }
+      setLastLang();
+    }
+    let lang;
+    if (keyboard.classList.contains('eng')){
+      lang = 'eng'
+      shiftKeyPress(e, keysEng, keysEngShift);
+    }else{
+      lang = 'rus';
+      shiftKeyPress(e, keysRus, keysRusShift);
+    }
+    return lang;
+  }
+
+  // Add 'active' class to the letters in css
+
+  function keyActive(e){
+    let index = keyCodes.indexOf(e.code);
+    if(e.type == 'keydown' && index != -1){
+      keyboard.children[index].classList.add('key_active');
```

Comment:
prefer to use constant variable instead of string literal multiple times

Reply:
Made `.key-active` as variable